components/compas-loading/src/compas-loading.ts

```diff
@@ -0,0 +1,15 @@
+import { html, LitElement, TemplateResult, property } from "lit-element";
+import "@material/mwc-list";
+import "@material/mwc-list/mwc-list-item";
+
+export class CompasLoadingElement extends LitElement {
```

Comment:
missing JSDoc here and for every prop like how it's done in `oscd-components` 
https://github.com/openscd/oscd-components/blob/main/components/oscd-action-icon/src/oscd-action-icon.ts

Reply:
Out of scope